contrib/packs/config.yaml

```diff
@@ -1,2 +1,30 @@
 ---
 pack_group: st2packs
+
+repositories:
+  default:
+    protocol: "ssh"
```

Comment:
Good suggestion, it would make things clearer for sure. There was a reason it ended up being like this. However I can't quite remember why... 

I'll look at refactoring this too.